How do we distribute money within the Astropy project?

hamogu opened the pull request:
The Astropy project has funding, currently through the Moore grant and we need to decide exactly what will be paid for with this money.

The PR introduces a *template* for internal calls for proposals that allow the community to decide how the money should be spent. On the one hand, it is important that all stakeholders in Astropy can have input, on the other hand, we need a process that generates as little extra work as possible. We want to spend our energy in improving the Astropy project (code, packages, community) and getting more resources (contributor, maintainers, funding, etc.) from the outside, not in competing with each on the money that we already have.

This PR makes a suggestion how the finance committee could issue an annual call asking to community to put forth the projects to be funded and how the community could decide on the distribution.

This proposal assumes that [APE0](https://github.com/astropy/astropy-APEs/pull/61) will be accepted to define voting members of the Astropy Project.

In this PR, we want to settle on the *general process* for distributing the money, the exact details of each internal call will have to be adjusted by the finance committee depending on what funding astropy has available in any given year (how much money? What are the detailed conditions in the grants?)

hamogu commented:
Please comment **before Dec 18th** if you have any suggestions!

hamogu commented on finance/proposal-calls/internal_template.md:
```diff
@@ -0,0 +1,56 @@
+
+At the second  deadline, the text for the proposals is frozen but comments are still allowed. Within 7 days of this deadline, the Finance Committee will review all proposals to ensure they are compatible with the terms of the currently available funding sources and should be included in the proposals to be voted on.
+
+After that, voting members of the Astropy Project vote on the proposals. The vote is held online with a secret ballot. Every voting member of the Astropy Project (as defined in the governance documents) gets X votes they can assign to one or more proposals. After the vote, proposals are ranked according to the number of votes.
```
That was the idea, so you could do 3 votes to proposal A (because it's really important to you) and 1 vote to C and D each (because those are good, certainly better than B, but not as good as A). But, as you say below, we many not need to sweat the details at this point.

astrofrog commented on finance/proposal-calls/internal_template.md:
```diff
@@ -0,0 +1,56 @@
+# Phase II funding proposals
+
+*This is a draft document for how the Interim Finance Committee envisions the process of distributing grant money (e.g. from the Moore grant) within the Astropy project. This is a general template and the finance committee will change the details for individual calls. In particular, actual dollar numbers and details of the grant and grant rules that limit the work that could be proposed will need ...
```
'dollar amounts' -> US centric wording, maybe generalize in case of non-US funding some day

astrofrog commented on finance/proposal-calls/internal_template.md:
```diff
@@ -0,0 +1,56 @@
+
+Proposals shall foster the Astropy Project goals of community, shared work, and cooperation and shall be proposed and executed in this manner.
+
+Total funding for this call is coming from (**link to grant here**) and XXX $ are available in total over XX months. 
```
from date X?

astrofrog commented on finance/proposal-calls/internal_template.md:
```diff
@@ -0,0 +1,56 @@
+
+- Source of money (e.g. Moore grant)
+- Specific area of proposal (e.g. Astropy may have a grant that can be used only for a specific purpose, e.g. only for community outreach)
+- Restrictions: Depending on who provided the money to astropy, it’s use might be restricted, e.g. only to contractors in certain countries
```
it's -> its

astrofrog commented on finance/proposal-calls/internal_template.md:
```diff
@@ -0,0 +1,56 @@
+- Restrictions: Depending on who provided the money to astropy, it’s use might be restricted, e.g. only to contractors in certain countries
+
+## Proposal process
+Proposals should be posted as pull requests to github.com/astropy/astropy-project/ with the initial title text "Phase II Proposal:".
```
Phase II Proposal -> isn't that specific to the Moore grant? If this is a template, should be a placeholder title?

hamogu replied:
Yes, but we don't have that many calls that we'll use cookiecutter to fill in this template. I anticipate absolutely minor re-writing to happen every time anyway.

astrofrog commented on finance/proposal-calls/internal_template.md:
```diff
@@ -0,0 +1,56 @@
+
+In either case, the Finance Committee will take care of the formal process of setting up statements of work, signing contracts, and reviewing invoices. Other funding strategies that we have not anticipated will also be considered.
+
+Note: The Finance Committee realizes that this process may lead to a number of parallel, possibly competing searches by different groups of people. We expect all searches to follow the code of conduct with respect to fairness, non-discimination etc. Complaints to ombdus@astropy, If these procedures are violated, funding for that specific project will not be granted. Proposals planning to do a sea ...
```
ombuds@astropy -> put proper email address?

astrofrog reviewed (commented):
Looks good to me overall! Just some small comments.

Note that some sources of funding may not allow 're-granting' so just to say that we should be careful that the source of funding allows this process (which I'm pretty sure the Moore funding does)

hamogu commented:
I've addressed all comments from the reviews. Thanks for reading through!
I'll now merge this document so that we can use it in the next phase of distributing funding. Any further changes can always be discussed in follow-up PRs.
